### ISSUE TITLE
coqbot not reporting all needed conditions for merge of PR

### ISSUE THREAD

**Issue by Alizter** [coq/bot#225]
It seems having a requested change in the PR doesn't get reported with the other conditions for merging. Have a look at this example:

https://github.com/coq/coq/pull/15852#issuecomment-1085495043

**Comment by Zimmi48** [comment]
Yes, this is expected since some checks require additional fetching (have a look at the code). We could do all the additional fetching eagerly to be able to report more condition failures if you want.

**Comment by Alizter** [comment]
I suppose the end goal here is to minimize the back-and-forth with the user and coqbot, so I see no harm in fetching data eagerly.
